pkg/agent/bakerapi.go

```diff
@@ -174,3 +175,7 @@ func findSIGImageConfig(sigConfig datamodel.SIGAzureEnvironmentSpecConfig, distr
 
 	return nil
 }
+
+func (agentBaker *agentBakerImpl) GetCachedVersionsOnVHD() (map[string]datamodel.ProcessedManifest, map[string]datamodel.ProcessedComponents, error) {
+	return datamodel.CachedFromManifest, datamodel.CachedFromComponents, nil
```

Comment:
we'd probably want to check whether these are actually instantiated and return an error if not, right now the error return value is unused